src/cgame/rocket/rocketCircleMenu.h

```diff
@@ -313,7 +313,7 @@ class RocketCircleMenu : public Rocket::Core::Element, public Rocket::Controls::
 					{
 						if ( GetChild( i ) == button )
 						{
-							parameters.Set( "index", button->GetAttribute< int >( "alienclass", 0 ) );
+							parameters.Set( "index", va( "%d", i - 1 ) );
```

Comment:
how about using the index if alienclass is not set as attribute?
and changing the alienclass to something more generic witch could be used for every cyrclemenu witch has hidden elements in its datasource.